cmd/index/job/deletion/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2019-2024 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"log"
```

Comment:
please use internal/log instead of official logger.

Reply:
Thanks for your review!
fix: https://github.com/vdaas/vald/commit/c7172a12d68e433c382553d7c5dbfbcc030f1e10